Tools/build/stable_abi.py

```diff
@@ -455,7 +453,7 @@ def do_unixy_check(manifest, args):
     okay &= _report_unexpected_items(
         extra_defs,
         'Some extra declarations were found in "Include/Python.h" '
-        + 'with Py_LIMITED_API:')
+        'with Py_LIMITED_API:')
```

Comment:
Here, I did want to point out that it's a single string and not a forgotten comma.
Could you point me to the reasoning behind this change?